src/stlinit.c

```diff
@@ -61,6 +61,7 @@ stl_initialize(stl_file *stl) {
   stl->stats.facets_malloced = 0;
   stl->stats.volume = -1.0;
   stl->stats.surface_area = -1.0;
+  memset(stl->stats.header, 0, sizeof(stl->stats.header));
```

Comment:
My C skills are very rusty. Does this nullify the entire string? Would setting the first character not be enough?

Reply:
Yes, it does. I chose this option to be always sure that the string is null terminated.